copyright

```diff
@@ -1377,12 +1377,14 @@ Files:
  images/ship/aerie*
  images/ship/bactrian*
  images/ship/mule*
+ images/ship/osprey*
  images/thumbnail/aerie*
  images/thumbnail/bactrian*
  images/thumbnail/mule*
+ images/thumbnail/osprey*
 Copyright: Saugia <https://github.com/Saugia>
 License: CC-BY-SA-4.0
-Comment: Derived from works by Michael Zahniser (under the same license).
+Comment: Derived from works by Michael Zahniser (under the same license) and detailed by Anarchist2.
```

Comment:
If this updated comment doesn't apply to all these images and is instead only for the Osprey then you should break the Osprey copyright off into its own section.

```
Files:
 images/ship/aerie*
 images/ship/bactrian*
 images/ship/mule*
 images/thumbnail/aerie*
 images/thumbnail/bactrian*
 images/thumbnail/mule*
Copyright: Saugia <https://github.com/Saugia>
License: CC-BY-SA-4.0
Comment: Derived from works by Michael Zahniser (under the same license).

Files:
 images/ship/osprey*
 images/thumbnail/osprey*
Copyright: Saugia <https://github.com/Saugia>
License: CC-BY-SA-4.0
Comment: Derived from works by Michael Zahniser (under the same license) and detailed by Anarchist2.
```